ui/src/app/settings/actions/config.js

```diff
@@ -13,4 +13,18 @@ config.fetch = () => {
   };
 };
 
+config.save = (name, value) => {
+  return {
+    type: "WEBSOCKET_SEND",
+    payload: {
+      actionType: "SAVE_CONFIG",
```

Comment:
Let's call this `UPDATE_CONFIG` to be consistent with the API.

Reply:
done